### ISSUE TITLE
Repeated code with exporting

### ISSUE THREAD

**Issue by falsandtru** [microsoft/TypeScript#12076]
This repeated code is added in a few weeks.

**TypeScript Version:**  master

**Code**

```ts
export class C {}
export namespace C {
  var c;
}
```

**Expected behavior:**

```js
"use strict";
var C = (function () {
    function C() {
    }
    return C;
}());
exports.C = C;
(function (C) {
    var c;
})(C = exports.C || (exports.C = {}));
```

**Actual behavior:** 

```js
"use strict";
var C = (function () {
    function C() {
    }
    return C;
}());
exports.C = C;
(function (C) {
    var c;
})(C = exports.C || (exports.C = {}));
exports.C = C; // <- here
```

cf:

```ts
export namespace C {
  var c;
}
```

```js
"use strict";
var C;
(function (C) {
    var c;
})(C = exports.C || (exports.C = {}));
```

**Comment by falsandtru** [comment]
I wrote the actual and expected instead of expected and actual. I swapped those two to fix them.

**Comment by falsandtru** [comment]
Oh... seems like I hacked GitHub while I was sleeping. I reported it to GitHub.

> @falsandtru falsandtru added the Bug label 3 hours ago
 @rbuckton rbuckton was assigned by falsandtru 3 hours ago
 @falsandtru falsandtru added this to the Future milestone 3 hours ago

**Comment by mhegazy** [comment]
:D I did that on your behalf. 

looks like github has been having issues today, and all my labels have been attributed to issue author.

**Comment by falsandtru** [comment]
I got a response from GitHub. They said Our team just pushed a fix for this. So it will be fixed soon.

**Comment by jakebailey** [comment]
Just checking out old bugs; this one was actually "fixed" very recently in 5.1 by #53901.

Though, I will say that this may not have really been a "bug" at all due to this sort of thing being required to emulate the fact that one can observe the exported value indirectly while it's partially evaluated (e.g. before the second declaration has had time to execute), as discussed in the PR.

But, the "unwanted" line noted in the issue is now gone so I think this can be closed.
